cov.py

```diff
@@ -31,33 +31,40 @@
 
 LOGGER = logging.getLogger()
 
-def collect_cov(log_dir, out, iss, testlist, batch_size, lsf_cmd, steps, \
-                opts, timeout, simulator, simulator_yaml, custom_target,
-                    target, stop_on_first_error):
+def collect_cov(log_dir, out, core, iss, testlist, batch_size, lsf_cmd, steps, \
+                opts, timeout, simulator, simulator_yaml, custom_target, \
+                isa, mabi, target, stop_on_first_error):
   """Collect functional coverage from the instruction trace
 
   Args:
-    log_dir          : ISS log directory
-    out              : Output directory
-    iss              : Instruction set simulator
-    testlist        : Testlist of the coverage test
-    batch_size       : Number of trace CSV to process per test
-    lsf_cmd          : LSF command used to run the instruction generator
-    steps            : csv:log to CSV, cov:sample coverage
-    opts             : Additional options to the instruction generator
-    timeout          : Timeout limit in seconds
-    simulator        : RTL simulator used to run
-    simulator_yaml   : RTL simulator configuration file in YAML format
-    custom_target    : Path for the custom target dir
-    target           : Predefined target
+    log_dir             : Trace log directory
+    out                 : Output directory
+    core                : RISC-V core to be used for analysis
+    iss                 : Instruction set simulator
+    testlist            : Testlist of the coverage test
+    batch_size          : Number of trace CSV to process per test
+    lsf_cmd             : LSF command used to run the instruction generator
+    steps               : csv:log to CSV, cov:sample coverage
+    opts                : Additional options to the instruction generator
+    timeout             : Timeout limit in seconds
+    simulator           : RTL simulator used to run
+    simulator_yaml      : RTL simulator configuration file in YAML format
+    custom_target       : Path for the custom target dir
+    isa                 : ISA variant
+    mabi                : MABI variant
+    target              : Predefined target
     stop_on_first_error : will end run on first error detected
   """
   cwd = os.path.dirname(os.path.realpath(__file__))
   log_list = []
   csv_list = []
-  trace_log = ("%s/%s_trace_log" % (out, iss))
   logging.info("Processing trace log under %s" % log_dir)
-  run_cmd("find %s -name \"*.log\" | sort > %s" % (log_dir, trace_log))
+  if core:
```

Comment:
Do you need this?

Reply:
I use this to properly name the output file `trace_log`, as well as filter out some extra files from the `find` command, I specify that logfiles should have `trace` somewhere in their name, as this directory also contains vcs simulation log files.

Reply:
I think you don't need log anymore, just need a list of CSV files?

Reply:
yes, you're right, I missed getting rid of this code after enabling full_trace by default. will fix this now